lbrynet/daemon/Daemon.py

```diff
@@ -2028,11 +2032,11 @@ def jsonrpc_publish(self, name, bid, metadata=None, file_path=None, fee=None, ti
         if bid <= 0.0:
             raise ValueError("Bid value must be greater than 0.0")
 
-        amt = yield self.session.wallet.get_max_usable_balance_for_claim(name)
-        if bid > amt:
+        balance = yield self.session.wallet.get_max_usable_balance_for_claim(name)
+        if bid > balance:
             raise InsufficientFundsError(
                 "Please lower the bid value, the maximum amount you can specify for this claim is {}"
-                .format(amt - MAX_UPDATE_FEE_ESTIMATE))
+                .format(balance - MAX_UPDATE_FEE_ESTIMATE))
```

Comment:
How come the condition check is `bid > balance` but in the error message it says that the max is `balance - MAX_UPDATE_FEE_ESTIMATE`. Should the condition be updated to be `bid > (balance - MAX_UPDATE_FEE_ESTIMATE)`?

Reply:
Since this is an estimate of the max fee that can be used, I put it in the message because one could technically update for a bid amount higher than that.
Should I instead put it in the condition check itself? This will _probably_ be changed after [this](https://github.com/lbryio/lbryum/pull/234) get merged though.